### PR TITLE
PCM-1726 

### PR DESCRIPTION
**Pull request by djhouseknecht**
removed package.json 'browser' field and pointed 'module' field to correct file name

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   },
   "main": "dist/npm/index.js",
   "module": "dist/es/index.js",
-  "browser": "dist/streaming-client.browser.js",
+  "web": "dist/streaming-client.browser.js",
```
Comment (djhouseknecht) [inline package.json:12]: `"web"` does not have any significance for build tools. I renamed it to this just so you can quickly see what the file name and purpose of the file is.

File: package.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
   },
   "main": "dist/npm/index.js",
   "module": "dist/es/index.js",
```
Comment (djhouseknecht) [inline package.json:11]: @jensengar can you confirm that this is the file that worked in Web-dir? I think it was (and I am going to test it more just to confirm).

Comment (jensengar) [reply package.json:11]: i will, but it might not be until tomorrow.

**Comment by djhouseknecht** [comment]
This will fix #117

**Review by zservies — APPROVED** [review]

**Pull request merged**